chartfx-chart/src/main/java/de/gsi/chart/plugins/EditAxis.java

```diff
@@ -359,7 +355,7 @@ private Pane getAutoRangeCheckBoxes(final Axis axis) {
             return boxMax;
         }
 
-        private final TextField getBoundField(final Axis axis, final boolean isLowerBound) {
+        private TextField getBoundField(final Axis axis, final boolean isLowerBound) {
```

Comment:
Codacy found an issue: [The method 'getBoundField(Axis, boolean)' has an NPath complexity of 320](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5764587)